File: unity/Assets/Scripts/Multiplayer/Players/EBanTime.cs
using UnityEngine;

namespace Michitai.Multiplayer.Players
{
    /// <summary>
    /// Ban duration options for player bans.
    /// Defines the time periods a player can be banned from the game.
    /// </summary>
    public enum EBanTime
    {
        /// <summary>1 hour ban duration.</summary>
        Hour,
        /// <summary>1 day ban duration.</summary>
        Day,
        /// <summary>1 week ban duration.</summary>
        Week,
        /// <summary>1 month ban duration.</summary>
        Month,
        /// <summary>3 months (quarter) ban duration.</summary>
        Quarter,
        /// <summary>1 year ban duration.</summary>
        Year,
        /// <summary>Permanent ban with no expiration.</summary>
        Forever
    }
}

File: java/src/main/java/com/michitai/multiplayer/players/EBanTime.java
package com.michitai.multiplayer.players;

/**
 * Ban duration options for player bans.
 * Defines the time periods a player can be banned from the game.
 */
public enum EBanTime {
    /** 1 hour ban duration. */
    HOUR,
    /** 1 day ban duration. */
    DAY,
    /** 1 week ban duration. */
    WEEK,
    /** 1 month ban duration. */
    MONTH,
    /** 3 months (quarter) ban duration. */
    QUARTER,
    /** 1 year ban duration. */
    YEAR,
    /** Permanent ban with no expiration. */
    FOREVER
}
